docs/guides/email/02-basic-email-system.md

```diff
@@ -57,10 +63,11 @@ zone "rockylinux.me" IN {
 
 !!! question 
 
-    **What is DNS zone?** A DNS zone is the specific portion of a DNS namespace that's hosted on a DNS server. A DNS zone contains resource records, and the DNS server responds to queries for records in that namespace. A DNS server can have multiple DNS zones. Simply put, a DNS zone is the equivalent of a book catalog.
+    **What is a DNS zone?** A DNS zone is aportion of the DNS namespace that's hosted on a DNS server. A DNS zone contains resource records, and a DNS server responds to queries for records in that namespace. A DNS server can have multiple DNS zones. Simply put, a DNS zone is analogous to a book catalog.
+
+First, initalize BIND:
```

Comment:
```suggestion
First, initialize BIND:
```

Reply:
@SergeCroise Above, you have "A DNS zone is aportion of..." If you are suggesting a change, it should be "A DNS zone is a portion of...", two words.